LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/opacity-valid-expected.txt

```diff
@@ -7,12 +7,12 @@ PASS e.style['opacity'] = "3" should set the property value
 PASS e.style['opacity'] = "-100%" should set the property value
 PASS e.style['opacity'] = "50%" should set the property value
 PASS e.style['opacity'] = "300%" should set the property value
-FAIL e.style['opacity'] = "clamp(50%, 0%, 70%)" should set the property value assert_equals: serialization should be canonical expected "calc(0.5)" but got "0.5"
-FAIL e.style['opacity'] = "clamp(50%, 80%, 70%)" should set the property value assert_equals: serialization should be canonical expected "calc(0.7)" but got "0.7"
-FAIL e.style['opacity'] = "clamp(50%, 60%, 70%)" should set the property value assert_equals: serialization should be canonical expected "calc(0.6)" but got "0.6"
-FAIL e.style['opacity'] = "min(50%, 0%)" should set the property value assert_equals: serialization should be canonical expected "calc(0)" but got "0"
-FAIL e.style['opacity'] = "min(0%, 50%)" should set the property value assert_equals: serialization should be canonical expected "calc(0)" but got "0"
-FAIL e.style['opacity'] = "max(50%, 0%)" should set the property value assert_equals: serialization should be canonical expected "calc(0.5)" but got "0.5"
-FAIL e.style['opacity'] = "max(0%, 50%)" should set the property value assert_equals: serialization should be canonical expected "calc(0.5)" but got "0.5"
-FAIL e.style['opacity'] = "min(-40%, 50%)" should set the property value assert_equals: serialization should be canonical expected "calc(-0.4)" but got "-0.4"
+PASS e.style['opacity'] = "clamp(50%, 0%, 70%)" should set the property value
```

Comment:
ah, so all css math function types should be serialized as "calc"

the description here is confusing. it should say:

```
PASS e.style['opacity'] = "clamp(50%, 0%, 70%)" should serialize as "calc(0.5)"
```